tests/unittest_nodes.py

```diff
@@ -155,8 +155,13 @@ class Language(metaclass=Natural):
         '''
 
         code_annotations = textwrap.dedent(code)
-        # pylint: disable=line-too-long
-        expected = 'def function(var: int):\n    nonlocal counter\n\n\nclass Language(metaclass=Natural):\n    """natural language"""'
+        expected = '''\
+def function(var: int):
+    nonlocal counter
+
+
+class Language(metaclass=Natural):
+    """natural language"""'''
```

Comment:
I would indent these lines here. For reference, see L149-L55.

Reply:
We're testing ``textwrap.dedent(code)`` so we're actually testing that the code line 149-155 become like the one here.

```
>       self.assertEqual(ast.as_string().strip(), expected)
E       AssertionError: 'def function(var: int):\n    nonlocal cou[68 chars]e"""' != '        def function(var: int):\n        [100 chars]e"""'
E       - def function(var: int):
E       +         def function(var: int):
E       ? ++++++++
E       -     nonlocal counter
E       +             nonlocal counter
E       ? ++++++++
E         
E         
E       - class Language(metaclass=Natural):
E       +         class Language(metaclass=Natural):
E       ? ++++++++
E       -     """natural language"""+             """natural language"""? ++++++++
```

Reply:
👍🏻 Got it